sentinel.conf

```diff
@@ -71,6 +71,22 @@ sentinel monitor mymaster 127.0.0.1 6379 2
 #
 # sentinel auth-pass mymaster MySUPER--secret-0123passw0rd
 
+# sentinel config-command <master-name> <command-name>
+# sentinel slaveof-command <master-name> <command-name>
```

Comment:
Well, I took auth-pass as a reference, which is not handled in `addReplySentinelRedisInstance`. If I understand it correctly, adding them there would change the response to client requests?